### DISCUSSION TITLE
feature request: allow me to disable them

### DISCUSSION THREAD

**Discussion by nektro** [community/community#9286] [Pull Requests] [answered]
you can turn issues, discussions, etc off but not pull requests. there are some projects where the model I would like to operate them with is only source-available, rather than allowing PRs from anyone.

**Comment by nektro** [comment, marked as answer]
seems to be a dupe of #8907

**Reply by grahamperrin** [reply]
Agreed. Can you delete this duplicate discussion? Or mark it as answered. Thanks.
